### PR TITLE
Feature/apvs 849/advanced search

### PR DESCRIPTION
**Pull request by pwright08**
Added remaining fields to advanced search page, and an end to end test.

Also fixes remaining part of #193.

## Checklist

- [x] Unit tests
- [ ] Code coverage checked
- [ ] Coding standards
- [ ] Error Handling
- [ ] Security/performance considered?
- [ ] Deployment changes considered?
- [ ] README updated

### CODE REVIEW

File: test/unit/routes/test-advanced-search.js [hunk 1]
```diff
@@ -70,21 +117,45 @@ describe('routes/index', function () {
...
-      }
+    it('should respond with a 200 and call data method', function () {
       getClaimListForAdvancedSearch.resolves({claims: [RETURNED_CLAIM], total: {Count: 1}})
       return supertest(app)
-        .get(`/advanced-search-results?reference=${searchCriteria.reference}&name=${searchCriteria.name}&draw=${draw}&start=${start}&length=${length}`)
+        .get(`/advanced-search-results?&draw=${draw}&start=${start}&length=${length}`)
         .expect(200)
         .expect(function (response) {
           expect(isCaseworkerStub.calledOnce).to.be.true
-          expect(getClaimListForAdvancedSearch.calledWith(searchCriteria, start, length), 'expected data method to be called with parameters').to.be.true
```
Comment (gdouglas1) [inline test/unit/routes/test-advanced-search.js]: Why was this line removed?  Should still be checking if the getClaimListForAdvancedSearch function is called with an empty searchCriteria object

Comment (pwright08) [inline test/unit/routes/test-advanced-search.js]: It was removed as I was testing the search criteria extraction and data call in the next test, but that's a good point - I'll add the check for the empty searchCriteria object in
